charts/rancher-cis-benchmark/0.1.1/questions.yaml

```diff
@@ -0,0 +1 @@
+rancher_min_version: 2.4.4-rc1
```

Comment:
Update this to 2.4.5-rc1. 

Do we need max version for 0.1.0 to 2.4.4? 

@prachidamle